inference-engine/tests/functional/plugin/cpu/single_layer_tests/mat_mul.cpp

```diff
@@ -32,10 +47,44 @@ class MatMulLayerCPUTest : public testing::WithParamInterface<MatMulLayerCPUTest
 
         std::tie(basicParamsSet, nodeType, fusingParams) = obj.param;
 
+        ElementType netType;
+        ElementType inType, outType;
+        ShapeRelatedParams shapeRelatedParams;
+        ngraph::helpers::InputLayerType secondaryInputType;
+        std::string targetDevice;
+        std::map<std::string, std::string> additionalConfig;
+        std::tie(shapeRelatedParams, netType, inType, outType, secondaryInputType, targetDevice, additionalConfig) =
+                basicParamsSet;
+
         std::ostringstream result;
         result << (nodeType == MatMulNodeType::MatMul ? "MatMul_" : "FullyConnected_");
-        result << LayerTestsDefinitions::MatMulTest::getTestCaseName(
-            testing::TestParamInfo<LayerTestsDefinitions::MatMulLayerTestParamsSet>(basicParamsSet, 0));
+        result << "IS=";
```

Comment:
we no need to print input dynamic shapes for static case

Reply:
We need to discuss this requirement. On the one hand it is pretty obvious that we do not need to print such info, on the other, `IS={}` may be an indicator as well. The former requires more complicated logic with testing `dynamicShape` part on emptiness, and, which is more important, it becomes even more complicated when we have to deal with multiple inputs.